corehq/apps/app_manager/suite_xml/generator.py

```diff
@@ -119,12 +121,11 @@ def media_resources(self):
         self.app.remove_unused_mappings()
         if self.app.multimedia_map is None:
             self.app.multimedia_map = {}
-        filter_multimedia = self.app.media_language_map and self.build_profile
+        filter_multimedia = self.build_profile and domain_has_privilege(self.app.domain, privileges.BUILD_PROFILES)
```

Comment:
is it possible for an app to have the privilege and then get those revoked?
In that case a previous version would have build profiles but not the privilege now?

Reply:
Yes, if the project space downgrades, the privilege will get revoked but the app will still have build profiles.

I ended up removing this code because adding more privilege checks wasn't the purpose of this PR and doing so was breaking tests (not really breaking them, it's just that patching `domain_has_privilege` is a pain).

Reply:
> Yes, if the project space downgrades, the privilege will get revoked but the app will still have build profiles.

And then they would not be able to use profiles for even a previous build which was made with profiles. Seems irrelevant now since the change is revoked but worth considering if we add in a follow up PR.

Reply:
Yeah, fair point. I don't really know what our position would be on downgraded projects - normally our stance is their existing apps should keep working in the field, but they can't update the paid features that app is using. So they shouldn't be able to make additional versions that use build profiles, but I'm not sure what approach we should take regarding old builds.